flox/core.py

```diff
@@ -1119,7 +1119,7 @@ def groupby_reduce(
     min_count: Optional[int] = None,
     split_out: int = 1,
     method: str = "map-reduce",
-    engine: str = "numpy",
+    engine: str = "flox",
```

Comment:
Can type these as `typing.Literal["flox", "numpy", "etc"]` if you want.

Reply:
Thanks!